.github/workflows/android-ci-pull.yml

```diff
@@ -15,6 +15,9 @@ on:
 jobs:
   build:
     runs-on: ubuntu-20.04
+    concurrency: 
+      group: ${{ github.ref }}
```

Comment:
I've used `group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}` elsewhere, to prevent this from cancelling things more widely than intended.

Reply:
Cool, let me try this then

Reply:
#330